frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -167,6 +167,11 @@ pods:
             dest: hadoop-2.6.0-cdh5.7.1/etc/hadoop/hdfs-site.xml
         env:
           FRAMEWORK_NAME: {{SERVICE_NAME}}
+        readiness-check:
```

Comment:
YAML allows comments.  Can you write a quick comment explaining what this readiness check does?